docs/API.md

```diff
@@ -51,19 +80,34 @@ Empty for now.
 - `Operation.type()` : string - Returns either `ClientSubscribing`, `ClientPublishing`, `ApplicationSubscribing`, `ApplicationPublishing`
 
 ### Channel
-- `Channel.path()` : string
+- `Channel.path()` : string (outcome of https://github.com/asyncapi/markdown-template/blob/master/components/Channels.js#L26)
```

Comment:
you mean name?

Reply:
@derber see https://github.com/jonaslagoni/parser-js/pull/1#discussion_r614096475

Reply:
In API I would first look for `id` or `name` and would never think of `path` 🤷🏼

Reply:
I agree path is not very suitable (I mentioned it earlier).
However, I see we have a problem with our domain language since, as per the spec, we talk about few other things but never about channel **name**.
If we believe this should be named as `name` or `id`, then it should be changed in the docs so it looks like:

![image](https://user-images.githubusercontent.com/1083296/115014733-03803300-9eb3-11eb-9a30-ceac3e26793d.png)

Reply:
If I saw `id` or `name` in the parser API I would have no idea what they represent in terms of channels 🤷 It does not explain anything about the underlying value and it has no parallel to the spec.

Reply:
We need to get an agreement here. 
I could vote for using `id` as standard for all or most of our entities if they share the same type. In this case, it's a bit special because the type is a [RFC 6570 URI](https://tools.ietf.org/html/rfc6570). However, we can say an ID it's just a `string` and all happy, right?

Reply:
Hmm... Maybe, I kinda disagree since I suspect that any developers which are using the API are gonna be confused about there being an `id` and what it entails. However, we can try to move forward with this, and get a third party to review it?

Reply:
I'll take a step back here. For more details look at https://github.com/jonaslagoni/parser-js/pull/1#discussion_r615785813

Go ahead 👍🏼